packages/babel-parser/src/parser/index.js

```diff
@@ -26,7 +26,7 @@ export default class Parser extends StatementParser {
     this.inModule = this.options.sourceType === "module";
     this.scope = new ScopeHandler(this.raise.bind(this), this.inModule);
     this.plugins = pluginsMap(this.options.plugins);
-    this.filename = options.sourceFilename;
+    this.filename = options.sourceFileName;
```

Comment:
Yeah I guess we wouldn't want to drop the other case until later (although I also don't expect it to be used often) as Logan said on Slack.

But I believe this is the only part that is breaking since the others are test cases, types, or local variables. We could do something like

```suggestion
    this.filename = options.sourceFilename || options.sourceFileName;
```

Reply:
I think this is a good idea, but it still doesn't seem to make the tests pass.